### DISCUSSION TITLE
typescript error, Property 'style' does not exist on Image

### DISCUSSION THREAD

**Discussion by dagda1** [ascorbic/unpic-img#439] [Q&A]
I use the style property to add the return of the `blurhashToGradientCssObject` but after upgrading to the latest version I get this error 

> Type '{ loading: any; layout: any; width: number; height: number; style: any; src: string; onLoad: () => void; }' is not assignable to type 'IntrinsicAttributes & (ImageProps & RefAttributes<HTMLImageElement>)'.
  Property 'style' does not exist on type '(IntrinsicAttributes & { src: string; width: number; height?: number | undefined; alt?: string | undefined; loading?: "eager" | "lazy" | undefined; decoding?: "async" | ... 2 more ... | undefined; ... 278 more ...; unstyled?: boolean | undefined; } & RefAttributes<...>) | ... 4 more ... | (IntrinsicAttributes & ... ...'.ts(2322)
  
  I can see in the types that the style property gets omitted with certain property combinations but I'm at a loss to understand which.
  
  WHy is style getting omitted?
  
  
<img width="1408" alt="Screenshot 2024-01-06 at 14 37 16" src="https://github.com/ascorbic/unpic-img/assets/118328/fd5a9ff5-637d-4ec5-9a7c-99ca60e140f0">

**Comment by ascorbic** [comment]
Hey. You should use `blurhashToCssGradientString`, and set it as the "background" prop.
